### PR TITLE
allow class including EPUB to intialize with extra arguments

### PR DESCRIPTION
**Pull request by skukx**
This is just an idea which I have mixed feelings about.  But it would be nice to instantiate a class with specific args.  For example:

```ruby
class Book
  include EPUB::Book::Features

  def initialize(args = {})
    @path_to_file = args[:path_to_file]
  end
end

options = {
  class: Book
  initialize_with: { path_to_file: '/path/to/file.epub' }
}

EPUB::Parser.parse('/path/to/file.epub', options)
```

### CODE REVIEW

File: lib/epub/parser.rb [hunk 1]
```diff
@@ -7,22 +7,22 @@ module EPUB
   class Parser
     class << self
       # Parse an EPUB file
-      # 
+      #
```
Comment (KitaitiMakoto) [inline lib/epub/parser.rb]: Could you keep white spaces?

**Comment by skukx** [comment]
Fixed whitespace changes

**Comment by KitaitiMakoto** [comment]
@skukx 
Thank you!

**Comment by KitaitiMakoto** [comment]
Released as version 0.2.7: https://rubygems.org/gems/epub-parser/versions/0.2.7

**Pull request merged**